src/lib/pages/workflows-with-new-search.svelte

```diff
@@ -218,7 +218,7 @@
   );
 
   $: {
-    if ($workflowFilters) {
+    if ($workflowFilters || (pageSelected && $updating)) {
```

Comment:
I think we want the store value (`$pageSelected`) here
```suggestion
    if ($workflowFilters || ($pageSelected && $updating)) {
```

Reply:
Ah good catch! Yeah we want `$pageSelected`, it's just a small optimization so that we only reset the checkboxes if they're checked in the first place.